docs/admin/federation/index.md

```diff
@@ -198,6 +255,98 @@ $ kubectl delete pods <pod-name> --namespace=kube-system
 
 We are now all set to start using federation.
 
+## Turn down
+
+In order to turn the federation control plane down run the following
```

Comment:
Remove "in order"; not necessary. "To turn down the federation control plane, run the following command:"